app/views/user_mailer/admin_hidden_work_notification.html.erb

```diff
@@ -1,12 +1,11 @@
 <% content_for :message do %>
-    <p>Dear <%= @user.default_pseud.byline %>,</p>
+    <p><%= t('user_mailer.admin_hidden_work_notification.hello', user: @user.default_pseud.byline) %></p>
 
-    <p>Your work <i><%= style_link(@work.title.html_safe, work_url(@work)) %></i> has been reported as being in violation of the Archive of Our Own's <%= style_link("Terms of Service", tos_url) %>.</p>
+    <p><%= t('user_mailer.admin_hidden_work_notification.html.part1', title: style_link(@work.title.html_safe, work_url(@work)), tos: style_link('user_mailer.admin_hidden_work_notification.tos', tos_url).html_safe).html_safe%> 
```

Comment:
There's a double `.html_safe` at the end of the line here -- is that needed to make the link work properly? 

Also, there's a space missing between the `.html_safe` and the `%>` at the very end: `.html_safe).html_safe%>`